packages/api-gen/src/commands/generate.ts

```diff
@@ -124,11 +124,14 @@ export async function generate(args: {
             schemaObject.type === "object" &&
             !!schemaObject?.properties?.translated
           ) {
+            schemaObject.required?.push("translated");
```

Comment:
this line probably should be only when we're adding `translated` property, so around line 160